src/Microsoft.VisualStudio.AppDesigner/Common/ShellUtil.vb

```diff
@@ -784,45 +784,6 @@ Namespace Microsoft.VisualStudio.Editors.AppDesCommon
             End If
         End Function
 
-
-        ''' <summary>
-        ''' Check if there's WCF import/generation error/warning. If there's one, force to display the error list page
```

Comment:
It's internal, so the only usage can come from those with InternalsVisibleTo access. I've searched the tree both via index and VSO and can find zero usage of this.